docs/src/main/sphinx/release/release-446.md

```diff
@@ -0,0 +1,67 @@
+# Release 446 (29 Apr 2024)
+
+## General
+
+* Improve performance of `INSERT` statements into partitioned tables when the
+  `retry_policy` configuration property is set to `TASK`. ({issue}`21661 `)
+* Reduce delay before stopping queries when the cluster runs out of memory. ({issue}`21719`)
+* Fix catalog names including `"` marks when creating them with `CREATE CATALOG`
```

Comment:
What do you mean by fixing the name? Disallow that? Or somehow change it?

Reply:
https://github.com/trinodb/trino/pull/21753#discussion_r1583848509